tools/prow-auto-bumper/main.go

```diff
@@ -65,6 +65,11 @@ var (
 	targetTime = time.Now().Add(-time.Hour * 7 * 24) // 7 days
 )
 
+// GHClientWrapper handles methods for github issues
+type GHClientWrapper struct {
```

Comment:
can we embed this instead of a new struct?
```
type wrapper struct {
ghutil.GHOperations
}
```

You can then call wrapper.<method>  directly and not have another redirection

Reply:
done